src/tracker-component.js

```diff
@@ -234,6 +251,9 @@ export const Tracker = (config? : Config = {}) => {
         JL.tracking(trackingConfig);
     }
     const trackers = {
+        getConfig: () => {
```

Comment:
What is this for? Testing?

Reply:
yeah, I was struggling w/ the tests because we had to use a bunch of roundabout ways to get the config, so thought I might just add a method to get it